docs/Doxyfile

```diff
@@ -0,0 +1,2622 @@
+# Doxyfile 1.9.1
+
+# This file describes the settings to be used by the documentation system
+# doxygen (www.doxygen.org) for a project.
+#
+# All text after a double hash (##) is considered a comment and is placed in
+# front of the TAG it is preceding.
+#
+# All text after a single hash (#) is considered a comment and will be ignored.
+# The format is:
+# TAG = value [value, ...]
+# For lists, items can also be appended using:
+# TAG += value [value, ...]
+# Values that contain spaces should be placed between quotes (\" \").
```

Comment:
This file is incredibly long. Could you maybe delete all the settings which are set to default anyway to keep this a bit tidier? And maybe reduce some of the comments? 😅

Reply:
I'll clean this up a bit.